tests/broker/broker_client/test_trading_routes.py

```diff
@@ -58,22 +130,34 @@ def test_get_open_position_for_account_with_asset_id(reqmock, client: BrokerClie
         f"{BaseURL.BROKER_SANDBOX}/v1/trading/accounts/{account_id}/positions/{asset_id}",
         text=f"""
         {{
-          "asset_id": "{asset_id}",
-          "symbol": "AAPL",
-          "exchange": "NASDAQ",
-          "asset_class": "us_equity",
-          "avg_entry_price": "100.0",
-          "qty": "5",
-          "side": "long",
-          "market_value": "600.0",
-          "cost_basis": "500.0",
-          "unrealized_pl": "100.0",
-          "unrealized_plpc": "0.20",
-          "unrealized_intraday_pl": "10.0",
-          "unrealized_intraday_plpc": "0.0084",
-          "current_price": "120.0",
-          "lastday_price": "119.0",
-          "change_today": "0.0084"
+            "asset_id": "{asset_id}",
+            "symbol": "AAPL",
+            "exchange": "NASDAQ",
+            "asset_class": "us_equity",
+            "avg_entry_price": "100.0",
+            "qty": "5",
+            "side": "long",
+            "market_value": "600.0",
+            "cost_basis": "500.0",
+            "unrealized_pl": "100.0",
+            "unrealized_plpc": "0.20",
+            "unrealized_intraday_pl": "10.0",
+            "unrealized_intraday_plpc": "0.0084",
+            "current_price": "120.0",
+            "lastday_price": "119.0",
+            "change_today": "0.0084",
+            "usd": {{
+                "avg_entry_price": "100.0",
+                "market_value": "600.0",
+                "cost_basis": "500.0",
+                "unrealized_pl": "100.0",
+                "unrealized_plpc": "0.20",
+                "unrealized_intraday_pl": "10.0",
+                "unrealized_intraday_plpc": "0.0084",
+                "current_price": "120.0",
+                "lastday_price": "119.0",
+                "change_today": "0.0084"
+            }}
```

Comment:
Just added this to all the relevant mock responses. Indentation seemed weird so I've re-indented everything hence why it looks like the whole response was updated in the diff 😅